concourse/client.py

```diff
@@ -221,7 +225,8 @@ def set_pipeline(self, name: str, pipeline_definition):
         headers = {'x-concourse-config-version': previous_version}
 
         url = self.routes.pipeline_cfg(name)
-        self._put(url, str(pipeline_definition), headers=headers)
+        response = self._put(url, str(pipeline_definition), headers=headers)
```

Comment:
could you check what the returned values for 'previous_version' (line 220/224) is in both cases (update vs create)?

I would somewhat prefer to use this to determine whether an update or a create occurred, provided the returned value reasonably suggests it could be used for this purpose (either being None or 0 or maybe 1). Just fearing this might be changed in future concourse versions (although this could also happen for the 'previous_version')